sys/riscv/riscv/pmap.c

```diff
@@ -566,7 +581,7 @@ pmap_bootstrap_l3(vm_offset_t l1pt, vm_offset_t va, vm_offset_t l3_start)
 	}
 
 	/* Clean the L2 page table */
-	memset((void *)l3_start, 0, l3pt - l3_start);
+	memset((void *)l3_start, 0, (vaddr_t)l3pt - (vaddr_t)l3_start);
```

Comment:
We should replace 'vaddr_t' with 'ptraddr_t' when bringing these changes into dev.

Reply:
Indeed, I missed this. I thought `vaddr_t` had permanently been removed and would fail to build?

Reply:
I've added `ptraddr_t`, but not widely adopted it to avoid churn during the current merges.

Reply:
Makes sense, I'll keep an eye out for `vaddr_t` then, as I was assuming the compiler would catch it.